helm-chart/renku-notebooks/templates/configmap.yaml

```diff
@@ -47,16 +47,12 @@ data:
 
     CURRENT_BRANCH=`git rev-parse --abbrev-ref HEAD`
     LOCAL_SHA=`git rev-parse --short HEAD`
-    REMOTE_SHA=`git rev-parse --short @{upstream}`
+    INITIAL_SHA="${CI_COMMIT_SHA:0:7}"
 
-    if [ -z "$REMOTE_SHA" ]; then
-      REMOTE_SHA="${LOCAL_SHA}"
-    fi
```

Comment:
Should we keep check (for `$INITIAL_SHA`) or there is a guarantee that `$CI_COMMIT_SHA` will always have the right value?

Reply:
I believe the `$CI_` variable are always there, those are the values inherited from the pod annotations. If not, the environments won't probably work correctly anyway :slightly_smiling_face: